loom/common/platform/platform.cpp

```diff
@@ -28,6 +28,8 @@ extern "C" {
 
 int platform_openURL(const char *url)
 {
+// Casting to int here is the right thing to do based on the func docs
+#pragma warning ( suppress: 4311, 4302 )
```

Comment:
Suppress is push, disable and pop in one statement!

Source: http://stackoverflow.com/a/18461418